harbor/README-v2.11.2.md

```diff
@@ -0,0 +1,16 @@
+| tlsCertificate.tlsSecretLabels	                               | {"managed-by": "vmware-vRegistry"} | 	The certificate that Tanzu Kubernetes Grid uses to install the Harbor CA as a trusted root on Tanzu Kubernetes Grid clusters.                                                                                                                 |
+| persistence.persistentVolumeClaim.registry.storageClass 	     | A storage policy name.             | 	A storage class that is used for the Harbor registry PVCs.                                                                                                                                                                                    |
+| persistence.persistentVolumeClaim.jobservice.jobLog.storageClass | A storage policy name.             | 	A storage class that is used for the Harbor jobservice PVCs.                                                                                                                                                                                  |
+| persistence.persistentVolumeClaim.database.storageClass	      | A storage policy name.             | 	A storage class that is used for the Harbor database PVCs.                                                                                                                                                                                    |
+| persistence.persistentVolumeClaim.redis.storageClass	         | A storage policy name.             | 	A storage class that is used for the Harbor redis PVCs.                                                                                                                                                                                       |
+| persistence.persistentVolumeClaim.trivy.storageClass	         | A storage policy name.             | 	A storage class that is used for Harbor trivy PVCs.                                                                                                                                                                                           |
+| enableNginxLoadBalancer 	                                     | true or false                      | Use nginx and LoadBalancer service to expose Harbor's endpoints when it's true.  This requires a loadBlalancer provider installed on Kubernetes cluster.  enableNginxLoadBalancer and enableContourHttpProxy can't be true at the same time.  When they are both set to false an Ingress will be created to expose Harbor's endpoints.	 |
+| enableContourHttpProxy	                                     | true or false                      | Use contour http proxy to expose Harbor's endpoints when it's true, enableNginxLoadBalancer and enableContourHttpProxy can't be true at the same time.  When they are both set to false an Ingress will be created to expose Harbor's endpoints.	                                           | 
```

Comment:
Suggest:
`Use contour http proxy to expose Harbor's endpoints when it's true, ` -> `When true, uses Contour's httpproxy resources to expose Harbor's endpoint. Ensure enableNginxLoadBalancer and enableContourHttpProxy are not both set true.`

Reply:
Updated